data/json/items/gunmod/mount.json

```diff
@@ -3,7 +3,7 @@
     "id": "grip_mount",
     "type": "GUNMOD",
     "name": { "str": "replaceable furniture kit" },
-    "description": "This is a kit consisting of various steel and plastic parts; when installed, it would permanently modify and partially replace some of the weapon's furniture so that it could be easily changed if needed.  This allows installing any kind of more advanced grips or other furniture.",
+    "description": "This is a kit consisting of various steel and plastic parts; when installed, it would permanently modify and partially replace some weapon's furniture so that it could be easily changed if needed.  This allows installing any kind of more advanced grips or other furniture.",
```

Comment:
Here, the sentence refers to "some of the furniture", not "some of the weapon". The edit changes the meaning

Reply:
Reverted, thank you